site/source/tutorials/004_user_interactions/index.md

```diff
@@ -91,27 +91,23 @@ Update the `WorkerProperties` interface to:
 
 {% include_codefile 'demo/finished/biz-e-corp/src/widgets/Worker.ts' lines:7-13 %}
 
-Now, we need to add the CSS selectors that will provide the rules for rendering this view's elements. Open up the `worker.css` file in `src/styles` and update it as follows:
-
-{% include_codefile 'demo/finished/biz-e-corp/src/styles/worker.css' lines:8-39 lang:css %}
-
-We also need to update the CSS selector for the front view, by changing the selector from `css.worker` to `css.workerFront`.
+Now, we need to update the CSS selectors that will provide the rules for rendering the updated widget. Open up the `worker.css` file and update it as follows:
```

Comment:
"Open up the `worker.css` file and update it as follows" -> "Edit `worker.css` as follows"